content/al/0006-produto-interno-ortogonalidade.md

```diff
@@ -0,0 +1,254 @@
+:::
+:::info[Simetria]
+$$ \langle x,x \rang \geqslant 0 $$ e $ \langle x,x \rang = 0 $ apenas quando $ x=0$
+:::
+
+A partir destas características fundamentais podese definir o conceito de espaço Euclidiano.
+
+:::info[Espaço Euclidiano]
+Espaço linear munido de munido de produto interno
+:::
+Num espaço euclidiano definem-se os seguintes conceitos:
+
+:::info[Norma]
+$$ \parallel x\parallel = \langle x,x \rang $$
+:::
+:::info[Distância]
+dist$$(u,v) = \parallel u-v\parallel $$
+:::
+
+### Matriz de Gram
+
+:::info[Matriz de Gram]
+Seja $W$ um espaço linear real (resp. complexo) munido de um produto interno e $$ B=(v*1,v_2,...,v_n)$$ uma base ordenada de $W$. A matriz $$G={[\langle v_i,v_j\rang]}*{(i,j=1,...,n)}$$ dos produtos internos dos vetores da base $B$ é dsignada por _matriz de Gram_ ou _matriz da métrica_, relativa à base $B$. A matriz $G$ verifica:
+
+1. $G$ é simétrica (respetivamente Hermitiana);
+2. $G$ é definida positiva, isto é, $$x^T_BGx_b>0$$ para todo $x \not = 0$ (resp. $ x^H_BGy_B>0$, para todo $x \not = 0$), ou seja, os valores próprios da matriz $G$ têm de ser todos positivos.
+
+Em relação à base $B$, o produto interno em $W$ escreve-se na forma
+
+$$ \langle x,y \rang = x^T_BGy_B$$
+onde $x_B$ e $y_B$ são respetivamente, os vetores de coodenadas de $x$ e $y$ na base $B$.
+:::
+
+### Desigualdade de Cauchy-Schwarz
+
+Num espaço euclidiano qualquer verifica-se que
+
+$$ \large{\dfrac{| \langle u,v\rang |}{\| u \| \|v \|}} \leqslant 1$$
+
+Esta desigualdade permite diretamente chegar à noção de ângulo entre vetores.
+
+:::info[Ângulo entre vetores]
+$$\large{\dfrac{ \langle u,v\rang }{\| u \| \|v \|}} =cos \theta, \theta \in [0, \pi] $$
+:::
+
+## Ortogonalidade
+
+Com todas as noções previamente discutidas, torna-se possível discutir ortogonalidade entre dois vetores.
+
+:::info[Definição]
+$ u \perp v<=> \langle u,v\rang=0 $
+:::
+
+Com a definição de ortogonalidade, pode-se concluir que o **Teorema de Pitágoras** é válido, tal que:
+
+Se $ u \perp v$ então
+
+$ \|u-v\|^2=\|u\|^2+\|v\|^2$Definição
+
+Com a ortogonalidade entre vetores definida sai a definição de conjunto ortogonal.
+
+:::info[Conjunto Ortogonal]
+$ S=\{v_1,v_2,...,v_n\}$ é ortogonal se os vetores de $S$ são ortogonais 2 a 2, isto é:
+
+$\langle v_i, v_j \rang= 0 $ para $i \not= j$
+
+:::
+
+De forma muito similar,
+:::info[Conjunto Ortonormado]
+$ S=\{v_1,v_2,...,v_n\}$ é ortonormado se os vetores de $S$ são ortogonais 2 a 2 e se a norma de todos os vetores fôr 1, isto é:
+
+$$\langle v_i, v_j \rang=$$ $$ \begin{cases}
+0 &\text{se } i \not = j \\
+1 &\text{se } i = j
+\end{cases} $$
+:::
+
+:::info[Proposição]
+Um conjunto ortogonal $ S=\{v_1,v_2,...,v_n\}$ que não contenha o vetor nulo é linearmente independente
+:::
+
+:::info[Projeção ortogonal]
+Num espaço lonear $W$ munido de um produto interno, a _projeção ortogonal_ do vetor $u \isin W$ sobre o vetor não nulo $v \isin W$. é definida por
+
+$$ proj_u v =\dfrac{ \langle u,v\rang }{\| u \|^2} $$
```

Comment:
```suggestion
$$
\op{proj}_u v =\dfrac{ \langle u,v\rang }{\| u \|^2}
$$
```

`\op{}` (custom shorthand for `\operatorname{}`) makes the underlying text not be italic (and is more semantically correct)